src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -372,7 +372,16 @@ Currently, webhooks must be configured per-project, which is unnecessary overhea
 
 **New Event Types**
 
-Sentry Integration apps expose a richer set of events that are not available via the legacy service hooks API. Currently, this includes Issue Created, Issue Resolved, Issue Ignored, and Issue Assigned. In the future, we’ll add more types of webhooks to Sentry Integration apps.
+Sentry Integration apps expose a richer set of events that are not available via the legacy service hooks API. Currently, this includes the following webhooks:
```

Comment:
We want to avoid Latin words and abbreviations

```suggestion
Sentry Integration apps expose a richer set of events that are not available using the legacy service hooks API. Currently, this includes the following webhooks:
```